crates/pop-parachains/src/new_parachain.rs

```diff
@@ -75,6 +78,20 @@ pub fn instantiate_standard_template(
 	Ok(tag)
 }
 
+pub fn instantiate_openzeppelin_template(
+	template: &Parachain,
+	target: &Path,
+	tag_version: Option<String>,
+) -> Result<Option<String>> {
+	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
+	let source = temp_dir.path();
+
+	let tag = Git::clone_and_degit(template.repository_url()?, source, tag_version)?;
```

Comment:
Any reason we are still cloning when we know its a GH repo and downloading and extracting from an archive is faster?

Reply:
Consistency with the rest of the parachain generations, I felt it was out of scope for this PR. It'll be refactored in a separate PR: https://github.com/r0gue-io/pop-cli/issues/312.